content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -228,7 +227,7 @@ rules:
 
 ## Normal User
 
-Few steps are required in order to get normal user to be able to authenticate and invoke API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate into the API call as the Certificate Header, or through the kubectl.
+There are a few steps are required in order to get normal user to be able to authenticate and invoke API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate into the API call as the Certificate Header, or through the kubectl.
```

Comment:
@EmilyM1 , These sentences could be reworded. Something like:

To set up a user for authentication to the API server, follow these steps:
1. Obtain a certificate issued by the Kubernetes cluster
1. Add the issued certificate as the certificate header to the API request or apply the certificate with `kubectl`